.github/workflows/pr-validation.yml

```
@@ -110,7 +110,7 @@ jobs:
 
   template-check:
     name: Validate PR Template
-    if: github.event.action != 'synchronize'
+    if: github.event.action != 'synchronize' and ${{ github.actor != 'renovate[bot]' }}
```

actionlint (1.7.12)

[error] if: condition "github.event.action != 'synchronize' and ${{ github.actor != 'renovate[bot]' }}" is always evaluated to true because extra characters are around ${{ }}

(if-cond)